custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -685,9 +685,16 @@ def fit(self, X, y, output_dir, class_order=None, row_weights=None, parameters=N
                         parameters=parameters,
                     )
 
-                    self._custom_task_class_instance.save(self._model_dir)
+                    try:
+                        self._custom_task_class_instance.save(self._model_dir)
+                    except Exception:
+                        raise DrumCommonException(
+                            "There appears to be an issue with your save hook in custom.py"
+                        )
                 except AttributeError:
-                    raise DrumCommonException("Fit method must be defined in your custom.py file")
+                    raise DrumCommonException(
+                        "There appears to be an issue with your fit hook in custom.py"
```

Comment:
We should leave the original message intact

Reply:
@jmbannon why? It's really misleading, because I would get that message often when a fit hook was actually defined and just had an error

Reply:
Hmmm, we should rethink this catch then. Maybe not even catch anything, so the stacktrace is intact

Reply:
Hmm, I don't know if I love that either. I'm going to combine the two comments as an interim step, i.e. something like "There appears to be an issue with your fit method OR it is missing entirely. Please verify that that the fit method in custom.py is present and runs without errors."

Reply:
Then I'm going to create a ticket to look into better logging and add this catch method in it, because we've talked several times about enhancing the logging anyway